.github/workflows/pyo3.yml

```
@@ -82,6 +82,14 @@ jobs:
             set -e
             pip3 install context-py --find-links dist --force-reinstall
             cd ./context-py && pytest
+      - name: Upload wasm package to S3
```

copy-pasta: We need to update the titles of each upload step 😅 

```suggestion
      - name: Upload wheel package to S3
```